editions/tw5.com/tiddlers/widgets/CheckboxWidget.tid

```diff
@@ -21,6 +22,7 @@ The content of the `<$checkbox>` widget is displayed within an HTML `<label>` el
 |tag |The name of the tag to which the checkbox is bound |
 |invertTag |When set to ''yes'', flips the tag binding logic so that the absence of the tag causes the checkbox to be checked  |
 |field |The name of the field to which the checkbox is bound |
+|fieldMode |<<.from-version "5.1.24">> Optional, turns on field list mode if set to "list". Requires //field// to be defined |
```

Comment:
what's the name of the default mode?

Reply:
I haven't given the default mode a name, partly because I couldn't come up with one. If the `fieldMode` (which might become `fieldType`) attribute is empty or anything other than "list" then it will work in default mode ("field mode" in the documentation).

Reply:
OK. I could think of a "binary", "single" mode as the default. .. So we have "binary" and "list" mode. In the future there may be more. IMO checkbox has 3 possible modes: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#additional_attributes ... indeterminate

Reply:
IMO indeterminate could be used for toggle, where neither `checked` nor `unchecked` value are part of the list